packages/contracts/src/contracts/current/protocol/Exchange/MixinExchangeCore.sol

```diff
@@ -116,6 +126,11 @@ contract MixinExchangeCore is
             return 0;
         }
 
+        if (order.salt < makerEpoch[order.maker]) { // Note that the 'salt' field is overloaded as a timestamp
```

Comment:
Let's re-assign `salt` to `timestamp` so that this comment is unnecessary:

```
uint256 timestamp = salt;
```

Are there any perf reasons not to do this?